packages/sanity/src/structure/panes/document/DocumentPaneProvider.tsx

```diff
@@ -415,8 +413,7 @@ export const DocumentPaneProvider = memo((props: DocumentPaneProviderProps) => {
 
       if (item.action === 'copy-document-url' && navigator) {
         telemetry.log(DocumentURLCopied)
-        const documentIntentLink = resolveIntentLink('edit', {id: documentId})
-        navigator.clipboard.writeText(window.location.origin + documentIntentLink)
+        navigator.clipboard.writeText(window.location.toString())
```

Comment:
can you add a comment here explaining why the intent link resolver wasn't used? I liked that implementation but after read the slack thread with jon's reasoning, i understand why. i think that explanation could be helpful here too